glmmTMB/tests/testthat/test-methods.R

```diff
@@ -654,18 +659,18 @@ test_that("weighted residuals", {
                      tolerance = 1e-6)
     }
 })
-
-test_that("bad inversion in vcov", {
-    skip_on_os(c("windows", "linux"))
-    d <- readRDS(system.file("test_data", "strengejacke_nasummary.rds",
-                             package = "glmmTMB"))
-    m <- glmmTMB(
-        QoL ~ time + age + x_tv_dm + x_tv_gm + z1_ti + z2_ti + (1 + time | ID) + (1 + x_tv_dm | ID),
-        data = d,
-        REML = TRUE
-    )
-    ## only fails on some platforms ... this is sufficient for now ... FIXME
-    if (getRversion() >= "4.3.0") {
-        expect_true(all(is.na(vcov(m)$cond)))
-    }
-})
+# This test started also giving a warning on os "mac".
```

Comment:
@bbolker do you think we should keep this? It seems like maybe it's more trouble than it's worth.

Reply:
it's certainly problematic ...  the original issues are [here](https://github.com/glmmTMB/glmmTMB/issues/376) and [here](https://github.com/glmmTMB/glmmTMB/issues/906); the latter seems like it might be a more "robustly bad" example (i.e., likely to fail more broadly ... ??). (The behaviour we're hoping to reproduce is a model that runs *without* warnings an errors, but where inverting the Hessian fails [which now gives NA values rather than an error] ...)  I just added an example in `misc/glmmTMB_GH906.R` -- but I think it would be OK, now that you've commented this test out here, to add something to the general issues list, and add this back once we've tested to see if that example is reliable across platforms/versions ...